src/components/molecules/NavBar/NavBar.scss

```diff
@@ -225,40 +225,6 @@ $border-radius: 28px;
   }
 }
 
-.back-map-btn {
-  position: fixed;
-  top: $navbar-height;
-  z-index: z(navbar-map-back-button);
-  left: 0px;
-  height: 40px;
-  width: auto;
-  cursor: pointer;
-  background-color: $secondary;
-  box-shadow: 0 10px 20px rgba($secondary, 0.2);
-  backdrop-filter: blur(8px);
-
-  border-radius: 0 0 16px 0;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 0 10px 0 0;
-  font-size: 0.9rem;
-
-  &:hover {
-    background-color: $lighter-intermediate-grey;
-    .back-icon {
-      opacity: 1;
-    }
-  }
-  .back-icon {
-    height: 22px;
-    width: 22px;
-    background-image: url(/assets/icons/slide-btn-icon.png);
-    background-size: 22px 22px;
-    opacity: 0.7;
-  }
-}
```

Comment:
[note] Presumably moved/refactored into `src/components/atoms/BackButton/BackButton.scss`

Reply:
Is it going to make it easier for you if I make these notes myself when creating the PR?

Reply:
Sometimes it's sort of obvious, but a lot of times it takes extra effort to figure out if it was deleted outright or moved/refactored somewhere else. So yeah, would help/make things easier.